seed/static/seed/partials/derived_columns_admin.html

```diff
@@ -0,0 +1,80 @@
+<div class="page_header_container">
+    <div class="page_header">
+        <div class="left page_action_container">
+            <a ui-sref="organizations" class="page_action"><i class="fa fa-chevron-left"></i> {$:: 'Organizations' | translate $}</a>
+        </div>
+        <div class="page_title">
+            <h1>{$:: org.name $}</h1>
+        </div>
+        <div class="right page_action_container">
+        </div>
+    </div>
+</div>
+
+<div class="section_nav_container">
+    <div class="section_nav" ng-include="::urls.static_url + 'seed/partials/accounts_nav.html'"></div>
+</div>
+
+<div class="section">
+    <div class="section_header_container">
+        <div class="section_header has_no_padding fixed_height_short">
+            <div class="section_action_container left_40">
+                <h2><i class="fa fa-calculator"></i> <span translate>Derived Columns</span></h2>
+            </div>
+        </div>
+    </div>
+    <div class="section_content_container">
+        <div class="section_content with_padding" style="margin-bottom:15px;">
+            <div class="data-quality-tab-container">
+                <ul class="nav nav-tabs" style="margin-bottom:1px;">
+                    <li ui-sref-active="active" heading="{$:: 'View by Property' | translate $}">
+                        <a ui-sref="organization_derived_columns(::{organization_id: org.id, inventory_type: 'properties'})" translate>View by Property</a>
+                    </li>
+                    <li ui-sref-active="active" heading="{$:: 'View by Tax Lot' | translate $}">
+                        <a ui-sref="organization_derived_columns(::{organization_id: org.id, inventory_type: 'taxlots'})" translate>View by Tax Lot</a>
+                    </li>
+                </ul>
+            </div>
+            <div class="table_list_container has_borders">
+                <table id="column-table" class="table has_no_btm_margin">
+                    <thead>
+                        <tr>
+                            <th class="text-left" ng-click="toggle_name_order_sort()">
+                                <span>Derived Column Name<i class="glyphicon glyphicon-sort"></i></span>
+                            </th>
+                            <th class="text-left">
+                                <span>Expression</span>
+                            </th>
+                            <th class="text-center" style="min-width: 120px; width: 120px;">
+                                <span>Actions</span>
+                            </th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                        <tr ng-if="derived_columns.length > 0" ng-repeat="derived_column in derived_columns">
+                            <td>
+                                {$:: derived_column.name $}
+                            </td>
+                            <td>
+                                {$:: derived_column.expression $}
```

Comment:
I'm going to come back and make the display of expressions prettier later (time letting). Just displays the expression with variable names in it for now